modules/sops/secrets-for-users/default.nix

```
@@ -36,6 +37,7 @@ in
         before = [ "systemd-sysusers.service" ];
         environment = cfg.environment;
         unitConfig.DefaultDependencies = "no";
+        path = cfg.age.plugins;
```

```suggestion
        path = lib.mkForce cfg.age.plugins;
```

this was causing errors in `nix flake check`, with
> [environment.](error: The option `systemd.services.sops-install-secrets-for-users.environment.PATH' has conflicting definition values)